docs/Documentation/Applications/gurobi.md

```diff
@@ -23,12 +23,12 @@ documentation](https://www.gurobi.com/documentation/).
 
 ## Available Modules
 
-| Kestrel         | Eagle           | Swift           |
+| Kestrel         | Swift           |
```

Comment:
Need to remove the third column in table ("|:---------------:|")

Reply:
Fixed in the most recent changes.